### PR TITLE
Updated account tile ui to be more customizable

### PR DESCRIPTION
**Pull request by joeyHarig**
- Added prop to prevent tiles from being clickable 
- Added prop to hide tile icons
- Added example of changes to docz

### CODE REVIEW

File: src/AccountTile/AccountTile-styled.js [hunk 1]
```diff
@@ -31,10 +31,14 @@ const StyledAccountTile = styled.div`
   transition: ${props => props.theme.transition};
   background-color: ${props =>
     props.open ? props.theme.palette.offWhite : props.theme.palette.white};
-  &:hover {
-    background-color: ${props => props.theme.palette.offWhite};
-    cursor: pointer;
-  }
+
+  ${props.clickable &&
```
Comment (codylawson) [inline src/AccountTile/AccountTile-styled.js:35]: You'll have to use a function here to reference props, similar to what we're doing on ln 32.

File: src/AccountTile/AccountTile-styled.js [hunk 2]
```diff
@@ -31,10 +31,14 @@ const StyledAccountTile = styled.div`
   transition: ${props => props.theme.transition};
   background-color: ${props =>
     props.open ? props.theme.palette.offWhite : props.theme.palette.white};
-  &:hover {
-    background-color: ${props => props.theme.palette.offWhite};
-    cursor: pointer;
-  }
+
+  ${props => props.clickable &&
+    css`
```
Comment (codylawson) [inline src/AccountTile/AccountTile-styled.js:36]: `css` is undefined here, you'll need to import it from styled components on line 13.

**Comment by joeyHarig** [comment]
@codylawson Everything should be ready to merge

**Review by codylawson — APPROVED** [review]
Looks good

**Pull request merged**